packages/shared/ReactFeatureFlags.js

```diff
@@ -37,7 +37,8 @@ export const warnAboutDeprecatedLifecycles = false;
 export const warnAboutLegacyContextAPI = false;
 
 // Gather advanced timing metrics for Profiler subtrees.
-export const enableProfilerTimer = __DEV__;
+export const enableProfilerTimer =
+  __DEV__ || process.env.REACT_ENABLE_PROFILING === true;
```

Comment:
Does it need to be on process.env, as opposed to another global?

My understanding is that this flag always gets replaced in build process. So we don’t actually let people use it, do we? Not unless they import our bundle directly. At that point we might as well call it `__PROFILE__` to better indicate its internal only.

Reply:
In case it’s not clear I mean that we didn’t change npm entry point so there’s no way external consumers can set this variable.

Reply:
I originally named it `__PROFILER__` but backed it out in 812beec because of Flow/Lint complaints. I don't feel strongly about that though. Can change it back.

Reply:
I'm mostly curious about your intention. What is your plan for people to use this in CJS mode?

Reply:
Intention wasn't to let people override this value at runtime, so you're right that using `process.env` was probably misleading or confusing.

> What is your plan for people to use this in CJS mode?

Haven't thought a lot about this, but I was assuming people would alias react (and renderer) imports for production builds if they wanted to use profiling capabilities.